scripts/azureml-assets/azureml/assets/model/model_utils.py

```diff
@@ -151,3 +153,51 @@ def prepare_model(spec_path, model_config, registry_name, temp_dir):
     except Exception as e:
         logger.log_error(f"prepare model failed for {spec_path}. Error {e}")
         return None, False
+
+
+def update_model_metadata(
+        model_name: str,
+        model_version: str,
+        spec_path: Path,
+        model_config: assets.ModelConfig,
+        registry_name: str,
+        update: AssetVersionUpdate = None):
```

Comment:
Please consider calling this function with just an AssetVersionUpdate object (instead of spec_path and model_config), which can either come from a deployment config file or be created using values from a spec.yaml file.

Reply:
Done. Thankyou Louie